cloudbank-v4/customer-helidon/src/main/resources/META-INF/microprofile-config.properties

```diff
@@ -17,4 +17,9 @@ javax.sql.DataSource.customer.password = Welcome12345
 # Enable Table Creation
 hibernate.hbm2ddl.auto=create
 hibernate.show_sql=true
-hibernate.format_sql=true
\ No newline at end of file
+hibernate.format_sql=true
+
+# Eureka configuration
+server.features.eureka.client.base-uri=http://eureka.eureka:8761/eureka
+server.features.eureka.instance.name=helidon-customer-service
+server.features.eureka.instance.hostName=helidon.helidon
```

Comment:
we have eureka configured with preferIp, so we might not need this?  if we do need to provide it, seems like the hostname provided would not be correct